public/js/extraFunctions.js

```diff
@@ -1011,54 +1025,119 @@ let initialize_tableData_dropdown = function() {
   }
 }
 
+// To add a tooltip to anything, follow these 3 easy steps
+// 1. Make sure the element allows for overflow
+// 2. Make sure the element's position is clearly defined as relative or absolute or something
+// 3. Give the element the attribute 'tooltip="loreum ipsum dolor sit amet"
+// If the tooltip's position needs to be readjusted manually, give it the attribute tooltip-margin.
+function setup_tooltips() {
+
+  // Checks if it already has a tooltip
+  for (const child of document.querySelectorAll('[tooltip]')) {
+
+    if (child.querySelector('.tooltiptext') === null) {
+
+      // Creates the tooltip
+      let node = document.createElement("SPAN")
```

Comment:
Use `const` instead of `let` unless you want to reassign a variable (const does not mean immutable) — this applies not only here but in other places of the code as well.